proposal/oscar97123/proposal.rst

```diff
@@ -0,0 +1,111 @@
+Proposal for Bitcoin trading bot
+===========================
+
+Bitcoin continues making headlines by breaking the currency rate to USD. The current price of 
+a single bitcoin dollar exceeds $50,000 US dollars continuously for the past month. Bitcoin is 
+a great hedging instrument like gold as it has limited supply and have pros like difficult 
+to manipulate as it is decentralized.
+
+The cash/currency that we used every day are all issued by the government, it might not seems 
+to be a big problem at a first glance. However, government-based currency can have 
+quantitative easing(QE) when they think it's needed. The problem is the newly generated money ("new money") 
+didn't go into the ordinary citizen's hand, instead, it goes to the enterprise.
+
+Even though the amount of money increased after QE, but, ordinary citizens saving remains the same.
+In other words, every time when QE being introduced, we all lost money. That's why it's needed to have 
+hedging strategery no matter it's bitcoin or not.
+
+Therefore, the reason that I choose this topic is because of hedging instrument, but also for speculation.
+Because the currency fluctuation is so high that a buy and a sell can generate gains, so it is worth to
+develop a trading bot specify for it.
+
+Basic information
+=================
+
+The GitHub repository (URL) hosting your term project. <TO-BE CREATED>
+
+The project aims to develop an auto trading bot specifily for bitcoin trading in order to gain profit 
+from buying the coin at low price but selling high. 
+
+
+Problem to solve
+================
+
+The idea of a trading bot is to automatically complete the buy and sell procedure based on the instructions given.
+The instruction might be set as a strictly by human or much smartly, by analysing the exchange insight of 
+recent market activities such as trading volume and Candlestick chart. Which, at most cases, is achieved by 
+adapting machine learning for analysing the past month's market activity. RNN / LSTM are the machine learning 
+model widely used.
+
+In this project, I'm going to applies the latter approach as it is much intelligent that the program itself 
+can have continous adjustment according to the recent market changes. Furthermore, I found the computation time 
+can be improved significantly since one of the Crypto trading bot "freqtrade" is written in Python language.
+
+The goals that I want to achieved for the project are as follow.
+
+|1. 5% calculation time improvement compares to exist trading bot "freqtrade". <The comparing platform TO-BE determined>
+
+|2. Reference others Cyprotocurrency market activity before making any buy/sell decision.
+
+|3. Users can choose simulated trading mode / actual trading mode though Telegram bot or though the python API.
+
+<<Math OR Algorithm>>
+|<TO-DO>
+
+
+Perspective users
+=================
+
+Any crypto traders and newbies to crypto trading.
+
+The user may use the software though Telegram bot or though python API.
```

Comment:
A telegram bot requires I/O which you need to describe if deciding to support it.

Reply:
May I reserved it as a nice-to-have feature since I think it depends on how well I catch up with the schedule?

If it is a yes, then in this case, do I still need to put a description of how the telegram bot works and how it handles the I/O in the "System architecture" part?

Reply:
Sure.  Please make it explicit in your proposal.